pkg/kubeconfig/kubeconfig.go

```diff
@@ -37,3 +37,15 @@ func NewMissingFileError(path string) error {
 func (m missingFileError) Error() string {
 	return fmt.Sprintf("kubeconfig file missing: path=%v", m.Path)
 }
+
+func ValidateKubeconfigPath(clusterName string) error {
```

Comment:
test?

Reply:
Added a test, to do so I did need to remove `*.kubeconfig` from our gitignore

Reply:
Tweaked the tests a bit so that we can keep this in gitignore